woocommerce/woocommerce
# Show feedback prompt only once

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR resolves the problem described in [this issue](https://github.com/woocommerce/woocommerce/issues/39167). We want to show the feedback prompts (to add and edit a product)only once in the legacy product experience.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a WooCommerce store with this branch and (**don't enable** the new product editor).
2. Under **WooCommerce > Settings > Advanced > Woo.com** verify that the `Enable tracking` option is `enabled`.  
3. Go to **Products > Add New**.
4. Add a product name and press `Publish`.
5. Verify the feedback prompt with the text `How easy was it to add a product?` is shown.

![Screenshot 2023-12-28 at 16 35 24](https://github.com/woocommerce/woocommerce/assets/1314156/7f7ce607-a480-44f6-a544-7e6793327086)

6. Edit that product and verify the feedback prompt with the text `How easy was it to edit your product?` is visible.

![Screenshot 2023-12-28 at 16 34 21](https://github.com/woocommerce/woocommerce/assets/1314156/347d3457-e0ec-4f80-b623-fe3164f5ec4e)

7. Add a change and save.
9. Verify that the feedback prompt is not being shown.
10. Repeat steps 3 and 4.
11. The feedback prompt shouldn't be shown.
12. If you want to test this again, you should set the option: `woocommerce_ces_shown_for_actions` to `a:0:{}` by using the WCA Test Helper under Tools. 



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
